java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2701,4 +2701,28 @@ private void writeListVector(UnionListWriter writer, int[] values) {
     }
     writer.endList();
   }
+
+  @Test
+  public void testVariableVectorGetEndOffset() {
+    try (final VarCharVector vector1 = new VarCharVector("v1", allocator);
+         final VarBinaryVector vector2 = new VarBinaryVector("v2", allocator)) {
+
+      setVector(vector1, STR1, null, STR2);
+      setVector(vector2, STR1, STR2, STR3);
+
+      assertEquals(0, vector1.getStartOffset(0));
+      assertEquals(6, vector1.getEndOffset(0));
```

Comment:
we can use STR1.length() to make it more readable?

Reply:
right, fixed.